platform/src/main/java/org/hillview/storage/GenericLogs.java

```diff
@@ -84,7 +84,9 @@ public ITable load() {
                     this.error("File " + this.filename + " is empty!");
                 if (this.genLog.schema.getColumnCount() == 0) {
                     this.genLog.schema.append(new ColumnDescription("Host", ContentsKind.String));
+                    reader.mark(1);
                     String line = reader.readLine();
+                    reader.reset();
```

Comment:
Isn't it simpler just to reuse the current value of the line in the while loop below?

Reply:
@mbudiu-vmw Done.